edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/authenticators/DeviceScopeTokenAuthenticator.cs

```diff
@@ -31,12 +31,12 @@ public class DeviceScopeTokenAuthenticator : DeviceScopeAuthenticator<ITokenCred
             this.edgeHubHostName = Preconditions.CheckNotNull(edgeHubHostName, nameof(edgeHubHostName));
         }
 
-        protected override (Option<string> deviceId, Option<string> moduleId) GetActorId(ITokenCredentials credentials)
-        {
-            // Parse the deviceId and moduleId out of the token's audience
-            SharedAccessSignature sharedAccessSignature = SharedAccessSignature.Parse(this.iothubHostName, credentials.Token);
-            (string hostName, string deviceId, Option<string> moduleId) = this.ParseAudience(sharedAccessSignature.Audience);
-            return (Option.Some(deviceId), moduleId);
+        protected override (Option<string> deviceId, Option<string> moduleId) GetActorId(ITokenCredentials credentials)
```

Comment:
what is the diff here? I cannot tell.

Reply:
Actually this entire file.

Reply:
yea the whitespacing got messed up again, been trying to fix it but git hasn't been cooperating